modin/experimental/engines/omnisci_on_ray/frame/omnisci_worker.py

```diff
@@ -17,12 +17,15 @@
 
 import pyarrow as pa
 import numpy as np
+from sys import platform
 
-prev = sys.getdlopenflags()
-sys.setdlopenflags(1 | 256)  # RTLD_LAZY+RTLD_GLOBAL
+if platform == "linux" or platform == "linux2":
```

Comment:
```suggestion
if sys.platform == "linux":
```
`linux2` is gone since Python 3.3 was released, we're not supporting anything older than 3.6 (or even 3.7) anyway :)

Reply:
Done